x/distribution/types/codec.go

```diff
@@ -36,10 +35,5 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 		&MsgDepositValidatorRewardsPool{},
 	)
 
-	registry.RegisterImplementations(
```

Comment:
should make a changelog entry that this proposal type is not supported with legacy content type

Reply:
I think this still supports legacy content type, we moved this registration to gov module codec.